aip/client-libraries/4222.md

```diff
@@ -154,6 +170,54 @@ If none of the routing parameters matched their respective fields, the routing h
 Much like URL parameters, if there is more than one key-value pair to be sent, the `&`
 character is used as the separator.
 
+### `path_template` syntax
+
+As seen in the above examples, the `path_template` can use a variety of symbols
+that are interpreted by code generators during conversion to regular expressions
+or non-regular expression matcher implementations. The `path_template` consists
+of segments delimited by the segment delimiter. The syntax for `path_template`
+is as follows:
+
+- The only acceptable segment delimiter is `/`.
+  - The last symbol in a `path_template` **may** be a delimiter - it will be 
+    ignored.
+- A segment **must** be of one of the following types:
+  - `*`: A single-segment wildcard. Corresponds to 1 or more non-`/` symbols.
+    The regex describing it is `[^\/]+`.
```

Comment:
nit: do not use backslash here, it's not a part of the regex syntax. It's only applicable in cases where the language itself uses `/  /` as delimiters.  The proper regex would be `[^/]+`.

Reply:
Done! Here and one other spot, thanks!